src/main/java/org/ethereum/beacon/discovery/pipeline/handler/MessageHandler.java

```diff
@@ -40,6 +40,9 @@ public void handle(Envelope envelope) {
     if (!HandlerUtil.requireField(Field.MESSAGE, envelope)) {
       return;
     }
+    if (!HandlerUtil.requireField(Field.SESSION, envelope)) {
+      return;
+    }
     if (!HandlerUtil.requireNodeRecord(envelope)) {
```

Comment:
This is covered by the `HandlerUtil.requireNodeRecord` below.  Could possibly rename that to `requireSessionWithNodeRecord` for clarity though.

Reply:
Ah, you're right. I missed that. Thanks, will fix for all instances 👍